flytepropeller/manager/shardstrategy/environment_test.go

```diff
@@ -0,0 +1,31 @@
+package shardstrategy
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestEnvironmentHashCode(t *testing.T) {
+	strategy1 := &EnvironmentShardStrategy{
```

Comment:
Nit: could we rename it?
`strategy1` -> `expectedStrategy`
`strategy2` -> `actualStrategy`

Reply:
sure!